src/components/addSourceWizard/compileAllApplicationComboOptions.js

```diff
@@ -13,6 +13,10 @@ export const descriptionMapper = (type, intl) =>
       defaultMessage: 'Analyze, forecast, and optimize your Red Hat OpenShift cluster costs in hybrid cloud environments.',
     }),
     [CLOUD_METER_APP_NAME]: <SubWatchDescription id={type.id} />,
+    [PROVISIONING_APP_NAME]: intl.formatMessage({
+      id: 'provisioning.sources.description',
+      defaultMessage: 'Builds and Launches images with custom content as Virtual Machines in hybrid cloud environments',
```

Comment:
I would go with:
"Build and launch images..."
Launch in this case is treated as a verb and not an app name. So my advice is to use lowercase. Also other descriptions on the page, don't use 3rd person. Let's be consistent. 

Malhar will say probably more.

Reply:
Thanks! Did that :)